docs/CHANGELOG.asciidoc

```diff
@@ -37,6 +38,266 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.8.0]]
+== {kib} 7.8.0
+
+See <<breaking-changes-7.8.0,breaking changes in 7.8>>.
+
+[float]
+[[enhancement-7.8.0]]
+=== Enhancements
+Alerting::
+* Adds query support to the Event Log {pull}62015[#62015]
+* Removes edit alert button from alerts list {pull}64643[#64643]
+* Implements actions server API for supporting preconfigured connectors {pull}62382[#62382]
+* Adds UI for pre-configured connectors. {pull}63074[#63074]
+APM::
+* Adda new Service map external icons {pull}63844[#63844]
+* Improves the selected node style {pull}64279[#64279]
+* Adds tooltip explaining Group ID {pull}60425[#60425]
+* Uses navigateToApp for infra/metrics/uptime links {pull}65712[#65712]
+* Observability real user monitoring solution layer {pull}64949[#64949]
+* Spans reparenting to support inferred spans {pull}63695[#63695]
+* Marks an Agent configuration as applied without etag attribute {pull}63967[#63967]
+* Agent remote config validation for Java agent configs {pull}63956[#63956]
+* Annotations API {pull}64796[#64796]
+* Adds Error Exception Type Column {pull}59596[#59596]
+* Service map anomaly indicators {pull}64718[#64718]
+Canvas::
+* Adds Top Menu {pull}59982[#59982]
+* Adds refresh and autoplay options to view menu {pull}64375[#64375]
+* Adds edit menu {pull}64738[#64738]
+* Reduces report generation time by re-using headless browser page in background {pull}63301[#63301]
+Dashboard::
+* Adds duplicate panel feature {pull}61367[#61367]
+* Return to dashboard after editing embeddable {pull}62865[#62865]
+* Deangularizes navbar, attempt nr. 2 {pull}61611[#61611]
+Lens and visualizations::
+* Triggers a filter action on click in datatable visualization {pull}63840[#63840]
+* Enables include/exclude in Terms agg for numeric fields {pull}59425[#59425]
+* Allows user to drag and select a subset of the timeline in the chart (aka brush interaction) {pull}62636[#62636]
+* Uses default {kib} palette for split series {pull}62241[#62241]
+* Removes duplicate refresh button from vis editor menu {pull}63633[#63633]
+* Binds all time fields to the time picker {pull}63874[#63874]
+* Uses Top 5 instead of Top 3 for first suggestion {pull}64726[#64726]
+* Adds pie and treemap charts {pull}55477[#55477]
+Machine Learning::
+* Wizards with dynamic model memory estimation {pull}60888[#60888]
+* Shows better file structure finder explanations {pull}62316[#62316]
+* Adds configurable file size to file data visualizer {pull}62752[#62752]
```

Comment:
There are a couple here related to the same change, so I recommend merging them:
```suggestion
* Adds configurable file size to file data visualizer {pull}62752[#62752], {pull}63502[#63502]
```